### PR TITLE
add initial project files

### PR DESCRIPTION
**Pull request by santiagonoguez**
including logging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "licensing-module",
+  "version": "1.0.0",
+  "description": "Licensing Module",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "eslint src && ./pre-build.sh && webpack --env.MODULE_VERSION=$(cat version) --config webpack.config.js && ./post-build.sh",
```
Comment (stulees) [inline package.json]: @santiagonoguez We don't need `--env.MODULE_VERSION=$(cat version)` in our module builds anymore.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MinifyPlugin = require("babel-minify-webpack-plugin");
+const nodeExternals = require("webpack-node-externals");
+const path = require("path");
+const UnzipsfxPlugin = require("unzipsfx-webpack-plugin");
+const ZipPlugin = require("zip-webpack-plugin");
+
+module.exports = env => {
```
Comment (stulees) [inline webpack.config.js]: @santiagonoguez We no longer need the `env.MODULE_VERSION` passed here (as mentioned also above). So this should now be 
```
module.exports = {
  entry: "./src/index.js",
  target: "node",
  externals: [nodeExternals()],
  output: {
    path: path.join(__dirname, "build", "licensing"),
    filename: "index.js"
  .....
```

Comment (santiagonoguez) [reply webpack.config.js]: @stulees removed the env.MODULE_VERSION reference from both files, thanks !

Comment (stulees) [reply webpack.config.js]: @santiagonoguez Still would remove the `env` param, so should like `module.exports = {`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "simple-mock": "^0.8.0",
+    "source-map-support": "^0.5.0",
+    "unzipsfx-webpack-plugin": "git://github.com/Rise-Vision/unzipsfx-webpack-plugin.git#v1.3.0",
+    "webpack": "^3.7.1",
+    "webpack-node-externals": "^1.6.0",
+    "zip-webpack-plugin": "^2.0.0"
+  },
+  "dependencies": {
+    "common-display-module": "git://github.com/Rise-Vision/common-display-module.git#v2.5.0",
+    "node-ipc": "^9.1.1",
```
Comment (andrecardoso) [inline package.json]: I think we don't need this dependency because `common-display-module` already has it.

File: src/logger.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable function-paren-newline */
+const common = require("common-display-module");
```
Comment (andrecardoso) [inline src/logger.js]: I'm seeing a logger class like this in every module. Maybe we should include it in `common-display-module` and have some `init`  method where the module passes its config

File: webpack.config.js (hunk 2 above)
Comment (santiagonoguez) [reply webpack.config.js]: @stulees  ohhh, right !!! removed !

File: package.json (hunk 3 above)
Comment (santiagonoguez) [reply package.json]: @andrecardoso removed !

File: src/logger.js (hunk 4 above)
Comment (santiagonoguez) [reply src/logger.js]: @andrecardoso I completely agree that repeated code across modules is a bad smell. Still there are some differences between the modules you mention, for example display-control includes functions to log display on and off, watchdog includes functions to log modules up and down, and so on. Also, some earlier modules use a different structure to define this logging access. Making this homogeneous may then require a proposal to define what's the best approach, and that clearly exceeds the scope of this card which is setting up an initial project for licensing. So I've created a card to propose that common logging code that modules can use and extend according to their needs: https://trello.com/c/UmX4J4cP/3799-3-refactor-common-logging-code-in-modules-to-common-display-module

Comment (andrecardoso) [reply src/logger.js]: Sounds good

**Pull request merged**